vital_checks.cpp

```diff
@@ -0,0 +1,42 @@
+// vital_checks.cpp
+#include "./vital_checks.h"
+#include "./warnings.h"
+
+// Constants for vital limits
+const float MAX_TEMPERATURE = 102.0;

```

Comment:
Nice naming! Can include the unit where needed. Temperature would need clarification (F or C), while the others don't need.

Reply:
Updated